force-app/main/default/objects/Thread__c/fields/HOT_DelPol_ToDeleteRecord__c.field-meta.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>HOT_DelPol_ToDeleteRecord__c</fullName>
+    <externalId>false</externalId>
+    <formula>IF(OR(ISPICKVAL(CRM_Thread_Type__c, &apos;HOT_BRUKER-FORMIDLER&apos;), ISPICKVAL(CRM_Thread_Type__c, &apos;HOT_BRUKER-TOLK&apos;), ISPICKVAL(CRM_Thread_Type__c, &apos;HOT_BESTILLER-FORMIDLER&apos;), (TODAY() - DATEVALUE(CreatedDate)) &gt; 365), true, false)</formula>
```

Comment:
Her ser det ut som det bør være en `AND` mellom sjekk på type og alder? Nå vil vel dette feltet være true for alle tråder av typene du har liste opp (uavhengig av alder), samt for alle tråder eldre en 365 dager (uavhengig av type).